tools/clang/unittests/HLSL/TestMain.cpp

```diff
@@ -21,6 +21,9 @@
 #endif
 #endif
 
+bool moduleSetup();
```

Comment:
TestMain.cpp is only included on non-windows builds. It's principle intent is to include a main() function which TAEF doesn't require since it loads test suites as dlls.

Reply:
ah Right.